src/style.css

```diff
@@ -1,3 +1,101 @@
 body {
   margin: 0;
+  font-size: 12px;
+
+  --select-color: #00acdc;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+p,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  margin: 0;
+}
```

Comment:
Do you have these elements on your page? If no, remove these styles as they are useless